composables/transaction/types.ts

```diff
@@ -241,7 +241,7 @@ export interface ActionMintCollection {
 
 export enum Collections {
```

Comment:
Why did you remove the set_max?

Reply:
`UPDATE_COLLECTION` is supposed to be the replacement

Reply:
`UPDATE_COLLECTION` is the replacement for that action

Reply:
yup I would keep it

Reply:
what about something like this [492f13b](https://github.com/kodadot/nft-gallery/pull/11171/commits/492f13b5557ea4b1f1cf55169ab24e258aa76d63), alternatively I can just a the old action `SET_MAX_SUPPLY` and just call that